themis/cred-offer.md

```diff
@@ -1,4 +1,4 @@
-# CRED_OFFER
+# Credential Offer
 
```

Comment:
We need both long names (e.g., Credential Offer) as well as msg types (e.g., CRED_OFFER) right?

Reply:
Currently there is nothing called a `CRED_OFFER` in the open source codebase atleast